styles/iconography/_styles.scss

```diff
@@ -47,6 +47,10 @@
         @include icon-small-block($fa-var-user);
     }
 
+    &.exclamation-triangle {
+        @include icon-small(before, right, $fa-var-exclamation-triangle);
```

Comment:
or .icon .exclamation-triage .small. But we are not following that pattern consistently...looks fine for now.